apps/site/src/app/apply/sections/Components/PasswordEye.tsx

```diff
@@ -0,0 +1,14 @@
+import { Eye, EyeOff } from "lucide-react";
+
+interface PasswordProps {
+	visible: boolean;
+	handler: React.MouseEventHandler<SVGElement>;
+}
+
+export default function PasswordEyeProps({ visible, handler }: PasswordProps) {
```

Comment:
I don't think this should be named with "Props" since it is a component

Reply:
Or actually I think @taesungh wants the password section remove all together, if I understand his comment correctly. So, you probably don't need to do this.